react/features/salesforce/actions.js

```diff
@@ -18,9 +20,12 @@ import { SalesforceLinkDialog } from './components';
  */
 export function showSalesforceNotification() {
     return (dispatch: Object, getState: Function) => {
-        const { salesforceUrl } = getState()['features/base/config'];
+        const conference = getCurrentConference(getState());
+        const salesforceEnabled = isSalesforceEnabled(getState());
+        const skipNotification = !salesforceEnabled
+            || conference.room.myroomjid.includes(conference.getBreakoutRooms().getComponentAddress());
```

Comment:
```suggestion
            || conference.getBreakoutRooms()?.isBreakoutRoom();
```

Reply:
You can simplify it like that.

Reply:
As it is in the isInBreakoutRoom selector, but this method returns undefined on the CONFERENCE_JOINED event.